fdbserver/include/fdbserver/workloads/workloads.actor.h

```diff
@@ -259,6 +259,12 @@ struct WorkloadFactory : IWorkloadFactory {
 	bool runInUntrustedClient;
 	WorkloadFactory(UntrustedMode runInUntrustedClient = UntrustedMode::False)
 	  : runInUntrustedClient(runInUntrustedClient) {
+		auto checkNameUnused = [](const std::string& name) -> bool {
+			auto f = factories();
+			return f.find(name) == f.end();
```

Comment:
Really a nit since I find `contains` more readable. 

```suggestion
			return !f.contains(name);
```